### ISSUE TITLE
cannot change environment variables after project creation

### ISSUE THREAD

**Issue by omercnet** [vercel/terraform-provider-vercel#129]
changing project environment variables after the resource has been created fails

```
    error: Error updating project: Could not upsert environment variables for project prj_....., unexpected error: bad_request - Invalid request: should be object.
```

while the [api docs](https://vercel.com/docs/rest-api/endpoints#create-one-or-more-environment-variables) say it's possible to create "one or more" if you look at the payload it accepts a single object and not an array

https://github.com/vercel/terraform-provider-vercel/blob/main/vercel/resource_project.go#L493

**Comment by dglsparsons** [comment]
Hi @omercnet, thanks for raising this - it sounds like a bug in the provider. 
What provider version are you using? Is this on latest?

**Comment by omercnet** [comment]
found the issue in the client, both requests for single and multiple were using the same api endpoint, attempting to use v10, although this payload is undocumented

**Comment by dglsparsons** [comment]
Thanks for submitting this fix - i've just pushed a tag for v0.15.1 containing the fix, so once CI has done it's thing, that will be released.

**Comment by omercnet** [comment]
thanks, and I'll update https://github.com/pulumiverse/pulumi-vercel accordingly ;)
